### PR TITLE
runn examples

### PR DESCRIPTION
**Pull request by atsushi-ishibashi**
refer: #344 

this is my image. If this differs from what you think, please give me feedback.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Runnable Examples
```
Comment (k1LoW) [inline examples/README.md:1]: I think it's a great idea to separate the runnable examples from the rest!

**Review by k1LoW — COMMENTED** [review]
GREAT!!

**Comment by atsushi-ishibashi** [comment]
I'd like to add link, which leads you to example readme, to the root README.
```
// README.md
### Examples

[details](./examples)
```

Whick place would you prefer? I think it would be prefer to have it on upper part.
For example, after quickstart or after usage.

**Comment by k1LoW** [comment]
@atsushi-ishibashi 

> I think it would be prefer to have it on upper part.
> For example, after quickstart or after usage.

I think that is a good position to start with.

**Comment by k1LoW** [comment]
GREAT!

**Comment by atsushi-ishibashi** [comment]
ok, ready for review👍 @k1LoW

**Review by k1LoW — APPROVED** [review]
Thank you!!

**Pull request merged**